mmv1/products/networkconnectivity/InternalRanges.yaml

```diff
@@ -0,0 +1,151 @@
+create_url: 'projects/{{project}}/locations/global/internalRanges?internalRangeId={{name}}'
+self_link: 'projects/{{project}}/locations/global/internalRanges/{{name}}'
+update_verb: :PATCH
+update_mask: true
+description:
+  'The internal range resource for IPAM operations within a VPC network. Used to represent a private address range along with behavioral characterstics of that range (its usage and peering behavior). Networking resources can link to this range if they are created as belonging to it.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1/projects.locations.internalRanges'
+  guides:
+    'Use internal ranges': 'https://cloud.google.com/vpc/docs/create-use-internal-ranges'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format:
+  [
+    "projects/{{project}}/locations/global/internalRanges/{{name}}",
+  ]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_internal_ranges_basic'
+    primary_resource_id: 'default'
+    vars:
+      internal_range_name: "basic"
+      network_name: "internal-ranges"
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_internal_ranges_automatic_reservation'
+    primary_resource_id: 'default'
+    vars:
+      internal_range_name: "automatic-reservation"
+      network_name: "internal-ranges"
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_internal_ranges_external_ranges'
+    primary_resource_id: 'default'
+    vars:
+      internal_range_name: "external-ranges"
+      network_name: "internal-ranges"
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_internal_ranges_reserve_with_overlap'
+    primary_resource_id: 'default'
+    vars:
+      internal_range_name: "overlap-range"
+      network_name: "internal-ranges"
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of the policy based route.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      User-defined labels.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      An optional description of this resource.
+  - !ruby/object:Api::Type::String
+    name: 'ipCidrRange'
```

Comment:
This needs `default_from_api: true` to cover the automatic assignment case!

Reply:
Done!